### PR TITLE
Give VPN users access to services running in assessment environments

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request gives VPN users access to services running in assessment environments.

## 💭 Motivation and context ##

Previously the list of such services included only our remote desktop gateway solution, which only needed a standard web port that was already allowed out, but now the list has grown to include some services that use other ports.

## 🧪 Testing ##

All `pre-commit` hooks pass.

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: egress_to_assessment_env_services_sg.tf [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Security group for assessment environment access
+resource "aws_security_group" "assessment_environment_services_access" {
+  vpc_id      = data.terraform_remote_state.networking.outputs.vpc.id
+  description = "Security group allowing access to assessment environment services"
```
Comment (dav3r) [inline egress_to_assessment_env_services_sg.tf]: Nitpicking, but don't we usually end our descriptions with a period?

**Review by dav3r — APPROVED** [review]
👍 👍

File: egress_to_assessment_env_services_sg.tf [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+# Access to assessment environment services (e.g., guacamole,
```
Comment (mcdonnnj) [inline egress_to_assessment_env_services_sg.tf]:
```suggestion
# Access to assessment environment services (e.g., Guacamole,
```

File: egress_to_assessment_env_services_sg.tf (hunk 1 above)
Comment (jsf9k) [reply egress_to_assessment_env_services_sg.tf]: See commit 6abc7c3.

**Review by mcdonnnj — APPROVED** [review]
Just one capitalization change but otherwise LGTM ✔

**Pull request merged**